tools/decode-status.py

```diff
@@ -178,7 +178,7 @@
     "USE_INA219","USE_SHT3X","USE_MHZ19","USE_TSL2561",
     "USE_SENSEAIR","USE_PMS5003","USE_MGS","USE_NOVA_SDS",
     "USE_SGP30","USE_SR04","USE_SDM120","USE_SI1145",
-    "USE_SDM630","USE_LM75AD","USE_APDS9960","USE_TM1638"
+    "USE_SDM630","USE_LM75AD","USE_APDS9960","USE_TM1638", "USE_HDC1080"
     ],[
```

Comment:
DON'T DO THIS. It breaks the program is it extents the array over it's limit.

If you are not sure what it does, do not make changes. Again, this is admin code that I will update once your driver is merged.